packages/kit/src/views/ReferFriends/pages/InviteReward/index.tsx

```
@@ -712,64 +719,71 @@ function Dashboard({
+                    <FiatValue
+                      fiatValue={hardwareSales.available?.[0]?.fiatValue}
+                    />
+                  ) : null}
+                  {showHardwarePendingFiat ? (
+                    <>
+                      <SizableText size="$bodyMd">{` + `}</SizableText>
+                      <NumberSizeableText
+                        formatter="value"
+                        size="$bodyMd"
+                        formatterOptions={{
+                          tokenSymbol: hardwareSales.pending?.[0]?.token?.symbol,
```

ESLint

[error] Insert `⏎···························`

(prettier/prettier)